index.html

```diff
@@ -1,47 +1,270 @@
-            <h2 class="foot">LET'S CONNECT</h2>
-            <ul class="social-media-icon">
-                <li>
-                    <a href="#" class="Linkdin"><img src="./images/linkedIn.png" alt="Linkedin"></a>
-                </li>
-                <li>
-                    <a href="#" class="twitter"><img src="./images/twitter.png" alt="Twitter"></a>
-                </li>
-                <li>
-                    <a href="#" class="M"> <img src="./images/M.png" alt="M"></a>
-                </li>
-                <li>
-                    <a href="#" class="cat"><img src="./images/cat.png" alt="Cat"></a>
-                </li>
-                <li>
-                    <a href="#" class="Bingo"><img src="./images/bingo.png" alt="Bingo"></a>
-                </li>
-            </ul>
+
+    <section class="intro">
+      <h2 class="myname">I'm Shaili Glad to see you!</h2>
+      <p class="about">
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+
+      <a href="#" class="social-media-icons">LET'S CONNECT</a>
+
+      <ul class="social-media-links">
+        <li>
+          <a
+            href="https://www.linkedin.com/feed/"
+            class="twitter-icon icon-right"
+          >
+            <img src="./images/twitter.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/feed/"
+            class="angelList-icon icon-right"
+          >
+            <img src="./images/bingo.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/feed/"
+            class="github-icon icon-right"
+          >
+            <img src="./images/cat.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/feed/"
+            class="linkdin-icon icon-right"
+          >
+            <img src="./images/M.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/feed/"
+            class="microsoft-icon icon-right"
+          >
+            <img src="./images/linkedIn.png" alt="" />
+          </a>
+        </li>
+      </ul>
     </section>
-</html>
\ No newline at end of file
+
+    <!-- ------Middle------section---- -->
+    <section class="works">
+      <section>
+        <img src="./images/snapshoot-portfolio.png" alt="" />
+        <h1 class="section-title">Tonic</h1>
+        <div class="section-about">
+          <a href="#" class="canopy">CANOPY</a>
+          <img src="./images/Counter.png" alt="" class="size" />
+          <h4 class="size back-end">Back End Dev</h4>
+          <img src="./images/Counter.png" alt="" class="size dot" />
+          <h4 class="year">2015</h4>
+        </div>
+        <p>
```

Comment:
- [ ] the content of each card in the works class should be aligned with flexbox.